libmeepgeom/Makefile.am

```diff
@@ -27,10 +27,14 @@ pw_source_ll_LDADD   = libmeepgeom.la $(MEEPLIBS)
 ring_ll_SOURCES = ring-ll.cpp
 ring_ll_LDADD   = libmeepgeom.la $(MEEPLIBS)
 
+array_slice_ll_SOURCES = array-slice-ll.cpp
+array_slice_ll_LDADD   = libmeepgeom.la $(MEEPLIBS)
+
 TESTS           = cyl-ellipsoid-ll
 
-noinst_PROGRAMS = bend-flux-ll
+noinst_PROGRAMS = bend-flux-ll array-slice-ll
```

Comment:
You're not actually running this test yet in `make check`, is that right?

Reply:
That's right.